### DISCUSSION TITLE
Developer Guide: Reset DB Command is not updated

### DISCUSSION THREAD

**Discussion by gelonatics** [block-crawler/block-crawler#15] [Q&A] [answered]
If you use this command **`block-crawler dev reset http://localhost:8000 db`** to reset db, you will have this error: 
**`ERROR: No such command 'reset'`**

The example command **`block-crawler dev reset http://localhost:8000 db`** should be **`block-crawler dev reset-db http://localhost:8000`**

**Comment by aenglander** [comment, marked as answer]
Thanks for posting this, I'll update the docs
